webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,182 @@
+  const [changedMeetingNoteIds, setChangedMeetingNoteIds] = useState<
+    EntityId[]
+  >([]);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [isSuccessIndicated, setIsSuccessIndicated] = useState(false);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+
+  const { data: meetingWithNotes, error } = useApiData<APIMeeting>({
+    deps: [changedMeetingNoteIds],
+    path: `/meetings/${meeting.id}`,
+    sendCredentials: true,
+  });
+  if (error) {
+    return <div>There was an error loading the meeting.</div>;
+  }
+  if (!meetingWithNotes) {
+    return null;
+  }
+  const currentParticipantId = meetingWithNotes.participants.find(
+    ({ principal_id }) => principal_id === principalId
+  )?.id;
+  let nextMeetingNoteSortOrder = 1;
+  for (let i = meetingWithNotes.meeting_notes.length - 1; i >= 0; i--) {
+    const meetingNote = meetingWithNotes.meeting_notes[i];
+    if (meetingNote.agenda_owning_participant_id === currentParticipantId) {
+      nextMeetingNoteSortOrder = meetingNote.sort_order + 1;
+      break;
+    }
+  }
+  const onSubmit: FormEventHandler = event => {
+    event.preventDefault();
+    setIsSavingMeetingNote(true);
+    fetch(`${process.env.REACT_APP_API_ORIGIN}/meetings/${meeting.id}/notes`, {
+      method: 'POST',
+      credentials: 'include',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        agenda_owning_participant_id: currentParticipantId,
+        note_text: meetingNoteText,
+        sort_order: nextMeetingNoteSortOrder,
+      }),
+    })
+      .then(res => res.json())
+      .then(({ data, error }) => {
+        setIsSavingMeetingNote(false);
+        setSaveMeetingNoteError(error || null);
+        if (data) {
+          setMeetingNoteText('');
+          setIsSuccessIndicated(true);
+          setTimeout(() => {
+            setIsSuccessIndicated(false);
+          }, 2000);
+          setChangedMeetingNoteIds([...changedMeetingNoteIds, data.id]);
+        }
+      })
+      .catch(error => {
+        setIsSavingMeetingNote(false);
+        setSaveMeetingNoteError(error);
+      });
+  };
+  return (
+    <Accordion
+      expanded={isExpanded}
+      onChange={(event, newIsExpanded) => setIsExpanded(newIsExpanded)}
+    >
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        sx={{
+          '&:hover': {
+            backgroundColor: 'grey.200',
+          },
+        }}
+      >
+        <MuiLink
+          component={ReactRouterLink}
+          to={`/meetings/${meeting.id}`}
+          underline="none"
+          sx={{ color: 'text.primary' }}
+          onClick={event => {
+            event.stopPropagation();
+          }}
+        >
+          <Typography>Meeting on {formatDate(meeting.starts_at)}</Typography>
```

Comment:
What does the `Typography` component do when no props are passed?

Reply:
The `Typography` actual wraps the text in a `<p>` tag and adds some styling to the text (to the default `body` MUI variant styles).  Otherwise, the string is just rendered as the content within `<a>` tags.